src/EyeTune/EyeTune.WindowsUniversal.Application/EyeTune.WindowsUniversal.Application.vcxproj.filters

```diff
@@ -221,6 +221,90 @@
     <None Include="$(MSBuildThisFileDirectory)..\..\..\presets\presets_eyetune\*.*" />
     <None Include="$(MSBuildThisFileDirectory)..\..\..\presets\presets_eyetune\*.*" />
     <None Include="$(MSBuildThisFileDirectory)..\..\..\presets\presets_eyetune\*.*" />
+    <None Include="$(MSBuildThisFileDirectory)..\..\..\presets\presets_eyetune\*.*" />
```

Comment:
lol what is this

Reply:
I was playing around with the EyeTune App example. This must've populated when I built it.